src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,19 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        ColorSupplier firstSupplier = new ColorSupplier();
+        ColorSupplier secondSupplier = new ColorSupplier();
+        ColorSupplier thirdSupplier = new ColorSupplier();
```

Comment:
We can call getRandomColor() method several times from single instance of service. No need in new service to call method
```suggestion
        ColorSupplier colorSupplier = new ColorSupplier();
```